README.md

```diff
@@ -1,93 +1,284 @@
+
+The conda Organization enforces this restriction by tracking the funding sources of each Steering Council member.
+
+### Shared Funding: Proximate, Ultimate, and In Between
+
+Funding can be multilayered and complicated.
+
+For example, Robert is a researcher in the Computer Science department at Western Michigan University and is working on a National Science Foundation (NSF) grant. For the purposes of the conda Organization, is Robert’s funder his CS department, the university, the specific grant, a division of NSF, the NSF, the US government, or all of the above?
+
+Meanwhile, across the world, Gouri works for GitHub, San-Yih works for LinkedIn, and João works on VS Code. Do they work for separate funders, or are they all funded by Microsoft?
+
+We use this guideline for determining which level(s) of funding to use for membership eligibility:
+
+> **When determining if funding is shared, choose the broadest level of funding where people might have a common set of goals because of their shared funding source.**
+
+In Robert’s situation, two levels are good candidates for determining shared funding.
+
+1. If Robert, or Robert’s department, is widely involved in many aspects of the conda ecosystem, then we may want to include his department in the list of Robert’s funders.
+2. If Robert is working on a grant that spans many universities then we may want to include the grant.
+
+In most circumstances we will not include the university as a funder because projects in the CS department, and in the Economics department, for example, are unlikely to have shared interests driven by both being at the same university. People funded by NSF, divisions of NSF, or the US government are also unlikely to have shared interests that are driven by their shared funding.
 
-For federated and incubating projects, the Project Team decides among themselves on how to add and/or remove members from the Project Team. Further, Project Teams for incubating projects are exempted from the requirement to have at least one Steering Council member. For a community maintained project, a petition for commit rights is made to the current members of the Project Team for that project. If this petition passes according to the voting rules for "Nominate new member of a Project Team" below, the petitioner is granted write access and added to the team.
+In the other example, Microsoft is a large company and these three largely unrelated parts of it are unlikely to have common goals based mainly on their common ultimate funder. However, a company that wanted to control the conda Organization could do so by enlisting people from across their organization, claim separate funding, and then capture the council. **In this case, the default guideline is to treat all parts of a company as having one funder.**
```

Comment:
How is "a company" determined here? Many companies have distinct legal entities. How does this handle investors? Eg. could a VC firm corner the market on conda by strategically investing in certain firms?

Do we expect members of the Council to reject grants or employment because it may cause them to no longer have voting rights? Asking maintainers to choose between their livelihood and the Conda project seems punitive and counter to a volunteer based organization.

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

Reply:
This does feel like a problem, but no simple solution immediately occurs to me.

Reply:
@CJ-Wright and @awwad: This guideline is suggested in the document:

> We use this guideline for determining which level(s) of funding to use for membership eligibility:
>
>> When determining if funding is shared, choose the broadest level of funding where people might have a common set of goals because of their shared funding source.

There is also a long discussion on proximate and ultimate funding, including long chains of funding.  So, I think the text covers the VC situation.  But the two of you don't think it does.  **What text could we add to cover this situation?**

Reply:
From @CJ-Wright 

> Do we expect members of the Council to reject grants or employment because it may cause them to no longer have voting rights? 

**The short answer is YES. This is exactly the tradeoff people will have to make.**  

If they switch to a funding source that already has 2 people on the Steering council, then one of those 3 people will have to lose their voting position.  That third person can still have a voice, just not a vote. For example, I don't have a vote, but as this PR shows, I still have a voice here.

This proposal seeks to strike a balance between encouraging involvement, preventing capture, and preferring simpler solutions over more complex ones.  (We did scope out a fractional voting system where each additional council member with the same funding would result in a fractional increase in the voting pool.  It would have partially addressed your concern, but it would have added a lot of complication to voting.)

This balance requires tradeoffs.  We could have different tradeoffs, but I don't see a solution that avoids them and still achieves the stated goals.

Open to suggestions (and I can probably find the fractional voting proposal if you want to see it).

Reply:
If folks are okay with the steering council voting to decide case by case on complex funding situations, then that makes sense enough to me.  Yes, the arrangement is a bit punitive, but Dave is right that there's a high complexity tradeoff to contend with.

Reply:
@CJ-Wright: We have added legal text ripped straight from the Python PEP about this.  That text was written by Van Lindberg when Python was dealing with this.

Reply:
This was NOT from a Python PEP but from the PSF by-laws: https://github.com/python/psf-bylaws/commit/4459f6eeee40f846540dccd655996b66365d6cd7.